doc/guides/connection_management.rst

```diff
@@ -274,24 +276,26 @@ dictionary specifying the synapse parameters in more detail.
     B = nest.Create('iaf_psc_alpha', n)
     nest.Connect(A, B, syn_spec='static_synapse')
 
-    syn_spec_dict = {'synapse_model': 'stdp_synapse', 'weight': 2.5, 'delay': 0.5})
+    syn_spec_dict = {'synapse_model': 'stdp_synapse', 
+                     'weight': 2.5, 'delay': 0.5}
     nest.Connect(A, B, syn_spec=syn_spec_dict)
 
-Specifying the synapse properties in a single dictionary or as
-:ref:`collocated synapses <collocated_synapses>` allows to draw
-synaptic parameters from random distributions and combine such
-parameters flexibly. In addition to the key ``synapse_model``, the
-dictionary can contain specifications for ``weight``, ``delay``,
-``receptor_type`` (see :ref:`receptor-types` for details) and
-parameters specific to the chosen synapse model. The specification of
+If synapse properties are given as a dictionary, this it must include 
+the key ``synapse_model``, and may additionally specify ``weight``, 
```

Comment:
You do not need to specify `synapse_model`, if you do not, the properties of `static_synapse` is changed. There is also a `this` too many in this sentence ;)

Reply:
@stinebuu I just revised the text, hope it is correct now :).